### ISSUE TITLE
Bunch of build errors

### ISSUE THREAD

**Issue by liamthedux124** [HamedMasafi/qt-rat#4]
could u guide me in the build process using visual studio code?

**Comment by HamedMasafi** [comment]
I have no idea about vscode. You can use QtCreator

**Comment by Dazmed707** [comment]
I install qtcreator
help me, step by step for compiling
video tutorial?

**Comment by HamedMasafi** [comment]
No there is no video tutorial. Just build the source and in case of fail put here the output

**Comment by Dazmed707** [comment]
What file do I open or in qtcreator in which option do I give to open the whole project to compile?
because I only get this
![image](https://user-images.githubusercontent.com/35184132/103976342-866f5000-5144-11eb-822f-e636f4413701.png)
 i open rat.pro

**Comment by HamedMasafi** [comment]
Are you new to qt? Then it's impossible for you to build this source

**Comment by Dazmed707** [comment]
I ask for help, I have only handled visual studio and compiled python software with pyinstaller.

**Comment by Ring0chaos** [comment]
i have git submodule init,but still have errors
![image](https://user-images.githubusercontent.com/38320742/165918072-5de725be-aea3-4935-ad26-20ed4346460a.png)
![image](https://user-images.githubusercontent.com/38320742/165918139-57839787-8baf-4b1e-b864-e4aee0f2dfc9.png)
please help me! or give me an binary file

**Comment by jelence** [comment]
yes,yes,i have the same problem with compile the source code,it appears i have no submodule.
#include <SharedObject>
![image](https://github.com/HamedMasafi/qt-rat/assets/9793998/bb9ea8b3-9242-46cb-8612-f412e15c51d4)
#include <ClientHub>
![image](https://github.com/HamedMasafi/qt-rat/assets/9793998/1abbdf28-faa9-4195-b2af-4c061b102374)
#include <Peer>
![image](https://github.com/HamedMasafi/qt-rat/assets/9793998/efd74ce2-5528-47d6-baf4-a026c4ae42e0)
